src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -614,7 +614,7 @@ void LogConfiguration::print_command_line_help(outputStream* out) {
   out->print_cr("Available log outputs:");
   out->print_cr(" stdout/stderr");
   out->print_cr(" file=<filename>");
-  out->print_cr("  If the filename contains %%p and/or %%t, they will expand to the JVM's PID and startup timestamp, respectively.");
+  out->print_cr("  If the filename contains %%p, %%t and/or %%hn, they will expand to the JVM's PID, startup timestamp and host name, respectively.");
```

Comment:
Why `%hn` rather than just `%h`?

Reply:
Because the log decorator for hostname is `hn`, so I think it's a good idea to choose the same.
Having two different "tags" for the same thing (hostname) within Unified Logging, would be more confusing.

Reply:
Ah I see. Too late to ask why UL chose that I guess :) I was thinking of the bash prompt \h for host name.